docs/source/release_notes/release-notes.rst

```diff
@@ -42,23 +44,71 @@ develop
+
+
+    *    - Type
+         - Change
+
+    *    - |userbreak| |improved|
+         - Added support for live-reloading sweep configurations.
+           ``TransactionManagers.create()`` methods now accept a Supplier of ``AtlasDbRuntimeConfig`` in addition to an ``AtlasDbConfig``.
+           If needed, the helper method `AtlasDbRuntimeConfig.defaultRuntimeConfig()` can be used to create a runtime config with the default values.
+           As part of this improvement, we made the sweep options of ``AtlasDbConfig`` unavailable.
+           The following options now **may not** be specified in the install config and must instead be specified in the runtime config:
+
+            +-----------------------------------+---------------------------+
+            | ``AtlasDbConfig``                 | ``AtlasDbRuntimeConfig``  |
+            +===================================+===========================+
+            | :strike:`enableSweep`             | **enabled**               |
+            +-----------------------------------+---------------------------+
+            | :strike:`sweepPauseMillis`        | **pauseMillis**           |
+            +-----------------------------------+---------------------------+
+            | :strike:`sweepReadLimit`          | **readLimit**             |
+            +-----------------------------------+---------------------------+
+            | :strike:`sweepCandidateBatchHint` | **candidateBatchHint**    |
+            +-----------------------------------+---------------------------+
+            | :strike:`sweepDeleteBatchHint`    | **deleteBatchHint**       |
+            +-----------------------------------+---------------------------+
+
+           Specifying any of the above install options will result in **AtlasDB failing to start**.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1976>`__)
+
     *    - |userbreak|
          - The Atomix algorithm implementation for the TimeLock server and the corresponding configurations have been removed.
            The default algorithm for ``TimeLockServer`` has been changed to Paxos.
            This should not affect users as Atomix should not have been used due to known bugs in the implementation.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2082>`__)
 
-    *    - |improved|
-         - The dropwizard independent implementation of the TimeLock server has been separated into a new project, ``timelock-impl``.
-           This should not affect users directly, unless they depended on classes from within the TimeLock server.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2081>`__)
-
     *    - |userbreak|
          - The previously deprecated RocksDBKVS has been removed.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1966>`__)
 
+    *    - |fixed| |improved|
+         - Sweep now dynamically adjusts the number of blocks, i.e., (cell, ts) pairs, across runs:
```

Comment:
I'd prefer for us not to use this terminology - a `block` is already defined as a fragment loaded by the stream store. Could we just use (cell, timestamp) pairs instead?